File: IMS-UI/ClientApp/src/app/IMS.Models/Employee/EmployeeOrdersResponse.ts
import { EmployeeOrderMapping } from "./EmployeeOrderMapping";
import { PagingInfo } from "../Shared/PagingInfo";
import { Response } from "../Response";

export class EmployeeOrdersResponse extends Response{
    employeeOrders: EmployeeOrderMapping[];
    pagingInfo : PagingInfo;
}

File: IMS-UI/IMS.Models/Employee/EmployeeOrdersResponse.cs
using IMS_UI.IMS.Models.Admin;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IMS_UI.IMS.Models
{
    public class EmployeeOrdersResponse : Response
    { 
        public List<EmployeeOrders> EmployeeRecentOrders { get; set; }
        public PagingInfo pagingInfo { get; set; }
    }
}
